src/commands/admin/activity.ts

```diff
@@ -1,41 +1,53 @@
-import { ActivityType, Message } from 'discord.js';
+import { ActivityType, ApplicationCommandOptionData, ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
 import { BOT, DB } from '@root/config';
-import { botMasterPerms } from '@lib/permissions';
+import { BOTMASTER_PERMS } from '@lib/permissions';
 import { Command } from '@lib/types/Command';
 
+const args = ['PLAYING', 'STREAMING', 'LISTENING', 'WATCHING', 'COMPETING'];
```

Comment:
The options can be sentence case (Playing, Streaming, etc.) for styling purposes. On line 36 the argument is capitalized anyway